dietpi/dietpi-services

```diff
@@ -216,7 +216,8 @@ Available services:
 			aSERVICE_NAME+=('nxserver')			# NoMachine Server
 			aSERVICE_NAME+=('xrdp') 			# XRDP Server
 			aSERVICE_NAME+=('amiberry')			# DietPi Amiberry service
-      		aSERVICE_NAME+=('unbound')  # Unbound DNS server, could be currently used by machine
+			aSERVICE_NAME+=('unbound')  # Unbound DNS server, could be currently used by machine
+			aSERVICE_NAME+=('tor')  # TOR relay should not go down normally
```

Comment:
What is the consequence when Tor goes down? Of course it breaks client access, but it does not affect the servers network access in any case, does it?

Generally, server services (used by clients that are not essential part of the server itself (network, remote control), are intended to be shut down during installs and maintenance tasks to minimise interference and resource usage. E.g. the just implemented Bitwarden_RS compiling and temporary Rust install require a lot of RAM and fail if that is not available. We applied some steps (check and in case increase swap size, limit concurrent compiling jobs), but if a high number of additional services is still running, that might not be enough anymore.

So as long as there is no forcing reason to keep Tor running, I suggest to let it usually handled by `dietpi-services`.